docs/Gallery.md

```diff
@@ -58,3 +61,179 @@ arrows depict the shift vector. Image credit: Alex Carpenter, CSUF
+
+Tutorial:
+
+- \ref tutorial_cce
+
+Publications:
+
+- Moxon et al. (2023). SpECTRE Cauchy-characteristic evolution system for rapid,
+  precise waveform extraction. \cite Moxon2021gbv
+- Moxon et al. (2020). Improved Cauchy-characteristic evolution system for
+  high-precision numerical relativity waveforms. \cite Moxon2020gha
+
+## Binary neutron star mergers
+
+SpECTRE can simulate merging neutron stars and other general-relativistic
+magneto-hydrodynamic (GRMHD) problems with dynamic gravity. Our DG-FD hybrid
+scheme accelerates smooth regions of the grid with high-order spectral methods
+(see \ref gallery_dgfd).
+
+Publications:
+
+- Deppe et al. (2024). Binary neutron star mergers using a discontinuous
+  Galerkin-finite difference hybrid method. \cite Deppe2024ckt
+
+\htmlonly
+<figure>
+<iframe src="https://drive.google.com/file/d/1kLx00pjDBmD49bpUcfHRcZNXR9jE8Sxb/preview" width="640" height="480" allow="autoplay" allowfullscreen></iframe>
+<figcaption>
+Simulation of two merging neutron stars. The colors show density contours.
+Video credit: Nils Vu, Caltech
+</figcaption>
+</figure>
+\endhtmlonly
+
+## Curved and moving meshes with control systems
+
+Our computational domains in SpECTRE are designed to adapt to the geometry of
+the problems we want to solve. They can be curved, e.g. to wrap around excision
+regions in binary black hole problems (see \ref gallery_bbh) or to resolve the
+wavezone in binary neutron star merger. They can also rotate and deform in time
+using control systems, which reactively adjust coordinate maps to track the
+position and shape of the black hole excisions or neutron stars.
+
+## Adaptive mesh refinement
+
+Our discontinuous Galerkin methods allow two types of mesh refinement: splitting
+elements in half along any dimension (h-refinement) or increasing their
+polynomial expansion order (p-refinement). The former allows us to distribute
+computational cost to supercomputers, while the latter allows us to use these
+resources efficiently by decreasing the numerical error exponentially with the
+number of grid points where the solution is smooth. Our adaptive mesh refinement
+technology decides which type of refinement to apply in each region of the
+domain.
+
+## DG-FD hybrid method {#gallery_dgfd}
+
+Our hydrodynamical simulations use a discontinuous Galerkin-finite difference
+(DG-FD) hybrid method: smooth regions of the simulation are evolved with an
+efficient DG scheme and non-smooth regions fall back to a robust FD method.
+Shocks and discontinuities on the grid are tracked with a troubled-cell
+indicator (TCI) to switch between DG and FD. This approach accelerates our
+simulations by reducing the computational resources spent on smooth regions of
+the grid, e.g. when evolving inspiral binary neutron stars and their
+gravitational radiation.
+
+Publications:
+
+- Deppe et al. (2024). Binary neutron star mergers using a discontinuous
```

Comment:
[Optional] You could cite https://arxiv.org/abs/2109.12033 since the conclusion was to use DG-FD methods

Reply:
fixed refs